opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/AbstractJettyTest.java

```diff
@@ -114,6 +119,34 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         }
     }
 
+    public static class AsyncServlet extends HttpServlet {
+
+        public static int ASYNC_SLEEP_TIME = 3000;
```

Comment:
Is there a test waiting 3000 seconds, or is it that high so that a timeout occurs? If so, then we should find another solution. Depending on a fixed time is never a good idea.